jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppContext.java

```diff
@@ -369,6 +369,11 @@ public void setClassLoader(ClassLoader classLoader)
             ((WebAppClassLoader)classLoader).setName(name);
     }
 
+    public ResourceFactory getResourceFactory()
```

Comment:
Interesting shortcut. Should it be moved to `Container` so that it's universally available?

Reply:
Hmm.
Maybe?
Future PR?